docs/tutorials/lrs/authentication/oidc.md

```diff
@@ -0,0 +1,172 @@
+# OpenID Connect authentication
+
+Ralph LRS also supports OpenID Connect on top of OAuth 2.0 for authentication and authorization.
+
+To enable OpenID Connect authentication mode, we should change the `RALPH_RUNSERVER_AUTH_BACKEND` environment variable to `oidc` and we should define the `RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI` environment variable with the identity provider's Issuer Identifier URI as follows:
+
+```bash
+RALPH_RUNSERVER_AUTH_BACKEND=oidc
+RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI=http://{provider_host}:{provider_port}/auth/realms/{realm_name}
+```
+
+This address must be accessible to the LRS on startup as it will perform OpenID Connect Discovery to retrieve public keys and other information about the OpenID Connect environment.
+
+It is also strongly recommended to set the optional `RALPH_RUNSERVER_AUTH_OIDC_AUDIENCE` environment variable to the origin address of Ralph LRS itself (e.g. "http://localhost:8100") to enable verification that a given token was issued specifically for that Ralph LRS.
+
+## Identity Providers
+
+OpenID Connect support is currently developed and tested against [Keycloak](https://www.keycloak.org/) but may work with other identity providers that implement the specification.
+
+## An example with Keycloak
+
+The [Learning analytics playground](https://github.com/openfun/learning-analytics-playground/) repository contains a Docker Compose file and configuration for a demo instance of Keycloak with a `ralph` client.
+
+First, we should stop the Ralph LRS server (if it's still running):
+```bash
+docker compose down
+```
+
+We can clone the `learning-analytics-playground` repository:
+```bash
+git clone git@github.com:openfun/learning-analytics-playground
+```
+
+Let's first create the network named `ralph`:
+```bash
+docker network create ralph
+```
+
+Then we can bootstrap the project:
+```bash
+cd learning-analytics-playground/
+make bootstrap
+```
+
+After a couple of minutes, the playground containers should be up and running.
+
+
+
+Create another docker compose file, let's call it `docker-compose.oidc.yml`, with the following content:
+```yaml title="docker-compose.oidc.yml" hl_lines="9-10 26-27 29-31"
+version: "3.9"
+
+services:
+
+  lrs:
+    image: fundocker/ralph:latest
+    environment:
+      RALPH_APP_DIR: /app/.ralph
+      RALPH_RUNSERVER_AUTH_BACKEND: oidc
+      RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI: http://learning-analytics-playground_keycloak_1:8080/auth/realms/fun-mooc
```

Comment:
```suggestion
      RALPH_RUNSERVER_AUTH_OIDC_ISSUER_URI: http://learning-analytics-playground-keycloak-1:8080/auth/realms/fun-mooc
```
The name of the keycloak container is different for me after `make bootstrap`. Does docker name the container differently in your case?

Reply:
You're right sorry!